db/views/reporting_overdue_patients_v01.sql

```diff
@@ -0,0 +1,89 @@
+SELECT DISTINCT ON (reporting_patient_states.month_date, reporting_patient_states.patient_id)
+  reporting_patient_states.month_date AS month_date,
```

Comment:
What do you think about adding month, year, quarter, month-string and quarter-string, to make it more convenient to use with metabase, and other ad-hoc reports and queries.

Reply:
These columns will take up a lot of space on the view, right? They are primarily redundant and can be realised by using a join on `reporting_months`.

Reply:
But I guess we want to prioritize convenience for the Metabase users.